src/views/category_panel.dot

```diff
@@ -69,7 +69,7 @@
                     {{= this.reviewsPartial({ reviews: grades, poi }) }}
                   {{?}}
 
-                  {{= this.minimalHourPanel.set(poi).render() }}
+                  {{= this.openingHourPartial(poi) }}
```

Comment:
This new partial removes a wrapping `div` (included by the old `Panel` function).  
As a consequence, the opening hours may be displayed on the same line as the reviews:

|master|d610f949a0c|
|-|-|
|![image](https://user-images.githubusercontent.com/4726554/66933748-cdea5780-f039-11e9-9f30-759d0635f3bf.png)|![image](https://user-images.githubusercontent.com/4726554/66933895-fc683280-f039-11e9-96ad-3ab66ff7cde3.png)|